lib/blobs/usbboot/LICENSE

```diff
@@ -1,30 +0,0 @@
-Copyright (c) 2006, Broadcom Corporation.
```

Comment:
This LICENSE shouldn't be removed, as it still applies to the `bootcode.bin` and `start.elf` files.

Reply:
Also, we might need to include a copy of the GPL2, as that'll apply to the kernel.img files we're including?

Reply:
Done!